mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -709,10 +735,10 @@ pub async fn nexus_create(
 
     if opened.is_ok() {
         nexus_list.push(ni);
-        Ok(name)
+        Ok(())
```

Comment:
If we remove the name from the return argument; there is no need to clone the name anymore  at 715.